### DISCUSSION TITLE
Desktop App 2.0 Release

### DISCUSSION THREAD

**Discussion by luongnh261** [lockerpm/.github#6] [Announcements]
Dear valued users, we are thrilled to announce the release of our brand-new desktop app! We are now inviting all users to participate in testing the desktop app version 2.0, as we are committed to bringing you an enhanced user experience with a range of improvements.

- Intuitive User Interface: We've revamped the interface to make it more user-friendly, allowing you to navigate and manage your passwords and other data effortlessly.

- Enhanced Stability: We've taken your feedback seriously and have worked diligently to address all the bugs and issues that were present in the earlier version. You can now expect a smoother and more reliable experience when using the desktop app.

- More packed-in features: The Quick share feature has been added to the new desktop app version, and we are implementing more features to include in the app, so stay tuned for more updates!

The new app is now ready at: https://locker.io/download. _For the Mac version, please follow the instructions attached below in this discussion to install the app._

We sincerely appreciate your continued support and valuable feedback. Your involvement in testing the desktop version 2.0 and providing us with your insights will play a crucial role in shaping our product in the future. We hope that the new desktop app will exceed your expectations, and we can't wait to hear your thoughts and feedback.

**Comment by luongnh261** [comment]
Due to a temporary issue in the distribution of the MacOS version, Mac users will need to follow the below steps to run the Locker Desktop App:

1. Download the app at https://locker.io/download#macos
2. Right-click on the app setup file, press and hold the Command key then click the Open option on the popup appear.

We are sorry for any inconvenience it may cause. We are actively working with Apple to resolve this issue. If you have any difficulties during the app installation process, please reach out to us at [contact@locker.io](mailto:contact@locker.io) for prompt assistance.

**Comment by baraach** [comment]
On the download page version 1.4.41 is available not version 2 
![image](https://github.com/lockerpm/.github/assets/89949942/dca1f527-e269-4509-97bc-930664587380)

**Reply by luongnh261** [reply]
Hi, this is a fault in versioning the app. We duly noted this issue and will fix it. 

Please ignore the version name. You can download the new app by clicking that link.

**Reply by luongnh261** [reply]
Hi, we are happy to inform you that this versioning issue has been fixed!

**Comment by Svec-Tomas** [comment]
Settings for MacOS Desktop never works for me. I have updated to newest version, originally was 2.0.3 now 2.0.11 (2.0.11.51).

Using MacBook Pro 14 M1 Pro, Sonoma 14.5

https://github.com/lockerpm/.github/assets/138758322/6edf799f-f985-474b-a0ef-e06777163b02

**Reply by luongnh261** [reply]
Sorry for this inconvenience. We have noticed this issue and are working on it. I will let you know as soon as it is fixed.

**Reply by luongnh261** [reply]
Hi, the issue was fixed in the newest version. Please uninstall this version and download the newest one at https://locker.io/passwords/download.
